### ISSUE TITLE
Request: VCO analog mode like VCV VCO

### ISSUE THREAD

**Issue by paulpiko** [bogaudio/BogaudioModules#170]
Is there any chance of getting an analog mode for the oscillators, similar to VCV's VCOs? Maybe in the context menu to avoid UI changes?

![image](https://user-images.githubusercontent.com/56526215/111936676-5536d700-8b1a-11eb-9e7a-111f00ec5672.png)

**Comment by mdemanett** [comment]
Yes, actually...while I wouldn't say it's coming soon, it's near the top of my list to do something like that.

**Comment by paulpiko** [comment]
Thanks Matt. Bogaudio modules among my favourites 🙂
